app/controllers/react_controller.rb

```diff
@@ -0,0 +1,3 @@
+class ReactController < ApplicationController
+  layout 'layouts/react_application'
```

Comment:
do we want to render this layout on each action?

Reply:
shouldn't there be only one action?